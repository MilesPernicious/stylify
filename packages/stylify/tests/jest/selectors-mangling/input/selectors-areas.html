{{ form_widget(form, { 'attr': {'class': 'color:blue font-size:1px'} }) }}
{{ form_widget(form, { 'attr': {"class": 'color:red font-size:2px'} }) }}

<div class="{active ? 'color:svelte1' : ''}">...</div>
<div class:color:svelte2={active}>...</div>
<div class:color:svelte3={!active}>...</div>
