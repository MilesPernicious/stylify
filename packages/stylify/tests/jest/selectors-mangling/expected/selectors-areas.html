{{ form_widget(form, { 'attr': {'class': 'b c'} }) }}
{{ form_widget(form, { 'attr': {"class": 'd e'} }) }}

<div class="{active ? 'a' : ''}">...</div>
<div class:f={active}>...</div>
<div class:g={!active}>...</div>
